src/installation/guides/chroot.md

```diff
@@ -168,6 +168,12 @@ Update the package manager and install `base-system`:
 # xbps-remove base-voidstrap
 ```
 
+If on aarch64, install a kernel package separately, for example:
```

Comment:
```suggestion
If on aarch64, it will be necessary to install a kernel package separately. For example, the `linux` package can be installed in conjunction with the `base-system` one.
```

And remove the example command. That said, I'm not sure this is under the correct section? It only applies to ROOTFS installs and not the other option (there is a command to install `base-system` further above in this guide).

Maybe it makes sense to add a small aarch64 intro that mentions the need to remember a kernel when doing things with `base-system`, instead of repeating the info in multiple places?

Reply:
there is no "in conjunction with", base-system on aarch64 does not install a kernel, that is the whole point

Reply:
I meant that in the sense that the command would be `xbps-install base-system linux`. If you can suggest better phrasing for it, I'd appreciate it.

Reply:
>For example, the `linux` package can be installed in addition to `base-system`.

should be concise and unambiguous

Reply:
Combined the suggestions from both of you.